e2e/settings/index.spec.ts

```diff
@@ -275,39 +275,36 @@ test("it is possible to edit variables", async ({ page }) => {
   ).toHaveValue(updatedType);
 });
 
-test("it is possible to handle broadcasts", async ({ page }) => {
-  const { broadcast } = await createBroadcasts(namespace);
+test("it is possible to update broadcasts", async ({ page }) => {
+  const { broadcast: broadcasts } = await createBroadcasts(namespace);
```

Comment:
I renamed `broadcast` to `broadcasts` in this context because it is an array, so it makes more sense to have a plural noun for it. Unfortunately, it is singular in the API response, but at least in this test this will improve readability.